django/db/models/expressions.py

```diff
@@ -749,6 +761,61 @@ def get_group_by_cols(self):
         return [self]
 
 
+class ExpressionList(Expression):
```

Comment:
`__repr__` and `__str__` are also implemented. I'm not sure if that's ideal, it made it easier in `Window` to write the `__str__` and `__repr__`-methods.